Makefile

```diff
@@ -27,7 +27,7 @@ pack-files:
 generate-proto: ## regenerate protos
 	@echo " > cloning protobuf from odpf/proton"
 	@rm -rf proton/
-	@git -c advice.detachedHead=false clone https://github.com/odpf/proton --depth 1 --quiet --branch main
+	@git -c advice.detachedHead=false clone https://github.com/odpf/proton --depth 1 --quiet --branch DBTCH-1024
```

Comment:
I don't see any PR in proton, can you share the URL here that we can review and get merged?

Reply:
the branch was there, I have created a PR here: https://github.com/odpf/proton/pull/14/

Reply:
updated